Tests/JWTTests/Helper/throwsError.swift

```diff
@@ -1,10 +1,12 @@
 @testable import JWT
 import XCTest
 
-func assert<E: Error>(_ expression: @autoclosure () throws -> Any,
+func assert<E: Error>(
```

Comment:
I believe what happened is that JWTError lost its conformance to Equatable when the `unknown(Error)` (thus with an associated value) was added. Xcode 8.3 beta 2 gave me a more helpful error message.